src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,23 @@
 package mate.academy.service;
 
+import mate.academy.model.User;
+
 public class AuthenticationService {
+
+    private User user;
```

Comment:
What is the reason to make User global class variable? I suggest to make it local variable in login method